builders/ethereum/precompiles/utility/non-specific.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Non-Network Specific Precompiles
+description: Learn how to use precompiled contracts, which are not specific to Ethereum or Moonbeam, yet are supported for use in your application.
+keywords: ethereum, moonbeam, StorageCleaner, ECRecoverPublicKey
+---
+
+# Non-Network Specific Precompiled Smart Contracts
+
+## Introduction {: #introduction }
+
+A precompiled contract, or precompile, is a set of programmed functionalities hard-coded into the blockchain client. Precompiles perform computationally heavy tasks, such as cryptographic processes like hashing. Moving these functionalities to the blockchain client serves the dual purpose of making the computation more efficient than using a traditional smart contract and ensuring everyone has access to the complete and accurate set of processes and algorithms required to operate correctly.
+
+Precompile functionality is bundled and shared under a smart contract address which allows interactions similar to those of a traditional smart contract. Some precompiled contracts are not specific to Ethereum or to Moonbeam, but are supported for use in your Moonbeam application. 
+
+The nonspecific precompiles currently included in this category include `StorageCleaner` and `ECRecoverPublicKey`. 
```

Comment:
This is missing SHA3FIPS256 & Dispatch

Reply:
Dispatch was removed from all three networks. I need to update on the Canonical Contracts page, etc. 

SHA3FIP is on the Eth Mainnet page. I forgot to move it over to this one. I'll take care of that.

Reply:
Done